rosa_getting_started/rosa-sts-getting-started-workflow.adoc

```diff
@@ -0,0 +1,22 @@
+include::modules/attributes-openshift-dedicated.adoc[]
+
+[id="rosa-sts-getting-started-workflow"]
+= Getting started using STS workflow
+:context: rosa-sts-getting-started-workflow
+
+toc::[]
+
+Follow this workflow to set up and access {product-title} (ROSA) clusters:
+
+. xref:../rosa_getting_started/rosa-sts-aws-prereqs.adoc#prerequisites[Perform the prerequisites].
```

Comment:
s/ Perform the prerequisites / Perform the AWS prerequisites.

Nermina's suggestion to use "Complete" is good. To that, I'm adding that it would be best to add AWS. This is because there is an entire section of AWS prerequisites that must be met before installing ROSA. And the AWS prerequisites need to be differentiated from the subsets of prereqs that exist in the individual task. WIsh I had thought of that before GA ;-)